docs/website/docs/dlt-ecosystem/destinations/qdrant.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Qdrant
+description: Qdrant is a high-performance vector search engine/database that can be used as a destination in dlt.
+keywords: [qdrant, vector database, destination, dlt]
+---
+
+# Qdrant
+
+[Qdrant](https://qdrant.tech/) is an open-source, high-performance vector search engine/database. It deploys as an API service, providing a search for the nearest high-dimensional vectors.
+This destination helps you load data into Qdrant from [dlt resources](../../general-usage/resource.md).
+
+## Setup Guide
+
+1. To use Qdrant as a destination, make sure `dlt` is installed with the `qdrant` extra:
+
+```bash
+pip install dlt[qdrant]
+```
+
+2. Next, configure the destination in the dlt secrets file. The file is located at `~/.dlt/secrets.toml` by default. Add the following section to the secrets file:
+
+```toml
+[destination.qdrant.credentials]
+location = "https://your-qdrant-url"
+api_key = "your-qdrant-api-key"
+```
+
+In this setup guide, we are using the [Qdrant Cloud](https://cloud.qdrant.io/) to get a hosted Qdrant instance and the [FastEmbed](https://github.com/qdrant/fastembed) package that is built into the [Qdrant client library](https://github.com/qdrant/qdrant-client) for generating embeddings.
+
+If no configuration options are provided, the default fallback will be `http://localhost:6333` with no API key.
+
+3. Define the source of the data. For starters, let's load some data from a simple data structure:
+
+```python
+import dlt
+from dlt.destinations.qdrant import qdrant_adapter
```

Comment:
I tested the example from the docs and it errors with this:
```
$ python qdrant_pipeline.py 
Traceback (most recent call last):
  File "qdrant_pipeline.py", line 26, in <module>
    qdrant_adapter(
TypeError: 'module' object is not callable
```

Where `qdrant_pipeline.py` is:

```python
import dlt
from dlt.destinations.qdrant import qdrant_adapter

movies = [
    {
        "title": "Blade Runner",
        "year": 1982,
    },
    {
        "title": "Ghost in the Shell",
        "year": 1995,
    },
    {
        "title": "The Matrix",
        "year": 1999,
    }
]

pipeline = dlt.pipeline(
    pipeline_name="movies",
    destination="qdrant",
    dataset_name="MoviesDataset",
)

info = pipeline.run(
    qdrant_adapter(
        movies,
        embed="title",
    )
)

print(info)
```

Reply:
https://github.com/dlt-hub/dlt/pull/724/commits/7980f386fac41053d49d7a6c351b64ef38499dba